crates/matrix-sdk-ui/src/timeline/event_item/content/message.rs

```diff
@@ -237,6 +237,13 @@ impl InReplyToDetails {
 
         InReplyToDetails { event_id, event: TimelineDetails::from_initial_value(event) }
     }
+
+    pub fn new_from_event(
+        event_id: OwnedEventId,
+        event: TimelineDetails<Box<RepliedToEvent>>,
+    ) -> InReplyToDetails {
```

Comment:
```suggestion
    ) -> Self {
```

You can also modify the return type of the `new` method.